distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -0,0 +1,90 @@
+==== Catalog Solr Embedded Provider
+
+The Solr Catalog Embedded Provider is an embedded, local Solr Server instance used in conjunction with an Apache Solr Server data store.
+It is a local instance that is a lightweight Catalog provider solution. However, it does not provide a Solr Admin GUI or a "REST-like HTTP/XML and JSON API." If that is necessary, see Standalone Solr Server.
+
+===== Installing Solr Catalog Embedded Provider
+
+. Navigate to the ${ddf-solr} application in the Admin console.
+. Under the *Features* tab, uninstall the `catalog-solr-external-provider` feature.
+. Install the `catalog-solr-embedded-provider`
+
+==== Configuring Embedded Solr Server and Solr Catalog Provider
+
+No configuration is necessary for the embedded Solr Server and the Solr Catalog Provider.
+The standard installation described above is sufficient.
+When the `catalog-solr-embedded-provider` feature is installed, it stores the Solr index files to `<DISTRIBUTION_INSTALLATION_DIRECTORY>/data/solr` by default.
+A user does not have to specify any parameters and the `catalog-solr-embedded-provider` feature contains all files necessary for Solr to start the server. 
+
+However, this component _can_ be configured to specify the directory to use for data storage. 
+
+The configurable properties for the SCP are accessed from the *Catalog Embedded Solr Catalog Provider* configurations in the ${admin-console}.
+
+[TIP]
+====
+The Embedded (Local) Solr Catalog Provider works on startup without any configuration because a local embedded Solr Server is automatically started and pre-configured.
+====
+
+include::{adoc-include}/_tables/ddf.catalog.source.solr.SolrCatalogProvider-table-contents.adoc[]
+
+Some notable features of the Solr Catalog Provider include:
+
+* Supports extensible metacards
+* Fast, simple contextual searching
+* Indexes XML Attributes as well as CDATA sections and XML text elements
+* Full XPath support.
+* Works with an embedded, local Solr Server (all-in-one Catalog)
+** No configuration necessary on a single-node distribution
+** Data directory of Solr indexes are configurable
+* Works with a standalone Solr Server
+
+.Embedded Solr Catalog Provider Pros and Cons
```

Comment:
Should the asterisks here be bullet points?
![image](https://cloud.githubusercontent.com/assets/8041246/18291304/f3381a18-743b-11e6-9910-7026bf5097cb.png)